crates/shared/src/http_solver/model.rs

```diff
@@ -188,10 +188,16 @@ impl SettledBatchAuctionModel {
     pub fn has_execution_plan(&self) -> bool {
         // Its a bit weird that we expect all entries to contain an execution plan. Could make
         // execution plan required and assert that the vector of execution updates is non-empty
-        self.amms
+
+        let amm_executions = self
+            .amms
             .values()
-            .flat_map(|u| &u.execution)
-            .all(|u| u.exec_plan.is_some())
+            .flat_map(|u| u.execution.iter().map(|e| &e.exec_plan));
+        let interaction_executions = self.interaction_data.iter().map(|i| &i.exec_plan);
```

Comment:
This turns out to be a breaking change. The otex solver was not providing an execution plan for its single interaction (it's marked optional on the struct) and therefore stopped being considered in shadow/barn mode and will basically break in prod if we release today.

Reply:
Thanks for noticing. Fixed in https://github.com/cowprotocol/services/pull/237